e2e_tests/standard/test_sync.py

```diff
@@ -17,134 +17,103 @@
 import json
 from pathlib import Path
 
+import pytest
 from click.testing import CliRunner
 
 import neptune.new as neptune
 from neptune.new.sync import sync
 
-from e2e_tests.base import BaseE2ETest, fake
-from e2e_tests.utils import DISABLE_SYSLOG_KWARGS, tmp_context
+from e2e_tests.base import BaseE2ETest, fake, AVAILABLE_CONTAINERS
+from e2e_tests.utils import (
+    DISABLE_SYSLOG_KWARGS,
+    initialize_container,
+    reinitialize_container,
+    tmp_context,
+)
 
 runner = CliRunner()
 
 
 class TestSync(BaseE2ETest):
-    SYNCHRONIZED_SYSID_RE = r"\w+/[\w-]+/([\w-]+)"
-
-    def test_sync_run(self, environment):
-        custom_run_id = "-".join((fake.word() for _ in range(3)))
+    SYNCHRONIZED_SYSID_RE = r"[\w-]+/[\w-]+/([\w-]+)"
```

Comment:
What was wrong with this RE?

Reply:
Wasn't matching workspace `e2e-tests`, for instance (which I was using ;)). This one is simply more robust.